dockerfiles/tests/Dockerfile

```diff
@@ -32,4 +32,5 @@ RUN find . -name \*.pyc -delete
 
 RUN chown -R splash:splash /app
 USER splash:splash
-ENTRYPOINT ["/app/dockerfiles/tests/runtests.sh"]
\ No newline at end of file
+ENTRYPOINT ["/app/dockerfiles/tests/runtests.sh"]
+CMD ["splash"]
```

Comment:
This was useful during tests to only run tests from `tests/test_proxy.py` locally by passing that path as an argument at the end of the `docker run` command, and I think it should be a safe change given it only affects tests, but reverting this change should not be a problem either, specially since it is technically unrelated to the patch.

Reply:
I think I better move it out to a separate pull request.